core/src/main/java/org/apache/brooklyn/core/effector/EffectorTasks.java

```diff
@@ -57,9 +58,14 @@
     private static final Logger log = LoggerFactory.getLogger(EffectorTasks.class);
     
     public interface EffectorTaskFactory<T> {
+        // TODO deprecate
```

Comment:
How does this relate to config of type Task? I thought that `EffectorTaskFactory` would be used in an entirely different context.